apps/rpc/src/trpc.ts

```diff
@@ -44,6 +44,28 @@ export const createContext = async (principal: Principal | null, context: Servic
           require(principal.affiliations.has(affiliation))
         }
       },
+      /**
+       * Require that the user is signed in and that the provided user id is the user's id.
+       */
+      requireMe(userId: UserId) {
+        this.requireSignIn()
+        invariant(principal !== null)
+        require(principal.subject === userId)
+      },
+      /**
+       * Requires either `requireMe` or `requireAffiliation` to be true.
+       *
+       * One of the following must be true:
+       * - The user is signed in and that the provided user id is the user's id.
+       * - The user is a member of at least one of the provided groups (or any group if empty).
+       */
+      requireMeOrAffiliation({ userId, affiliations }: { userId: UserId; affiliations: Affiliation[] }) {
```

Comment:
I would prefer to pass two arguments instead here. It's rather easy to distinguish a scalar from an array value